### PR TITLE
Hide datepicker on tab out

### PR DESCRIPTION
**Pull request by jdtibbs**
Restore the behavior introduced in this [pull request](https://github.com/qodesmith/datepicker/pull/2)

### CODE REVIEW

File: src/datepicker.js [hunk 1]
```diff
@@ -1282,6 +1282,9 @@ function oneHandler(e) {
 
     // Hide all other instances.
     hideOtherPickers(instance)
+  } else if (type === 'keydown' && e.keyCode === 9 && instance) {
```
Comment (qodesmith) [inline src/datepicker.js:1285]: This is great. A few things...

1. Can you use a top level variable to store the keycode? And use `e.which || e.keyCode` for compatibility.
2. Can you refactor the lines below this which also check for keyCodes? I'll leave comments on each where you can replace that logic with the variable.

Thanks for this!

Comment (qodesmith) [reply src/datepicker.js:1285]: It won't let me leave comments on the other lines, but as of this commit they are lines 1293 and 1297.

**Comment by jdtibbs** [comment]
Sorry I'm new to this pull request to a forked repo. Thank you for the review and considering this feature.

File: src/datepicker.js [hunk 2]
```diff
@@ -1196,8 +1196,10 @@ function oneHandler(e) {
   */
   if (e.__qs_shadow_dom) return
 
+  var keyCode = e.keyCode
```
Comment (qodesmith) [inline src/datepicker.js:1199]: One last thing... can you change this to: `var keyCode = e.which || e.keyCode`? Then you can can refactor the `if` statements to check a single variable. Once you make the change, I'll merge.

Thank you for the PR!

**Comment by qodesmith** [comment]
Thanks for the PR. I appreciate your work on this!

File: src/datepicker.js (hunk 2 above)
Comment (jdtibbs) [reply src/datepicker.js]: Good point.

**Comment by jdtibbs** [comment]
Thank you for creating this nice date picker. Exactly what I need, and no more.

**Pull request merged**